### PR TITLE
use github actions bot user in benchmark workflow

### LINKED ISSUE #1318: use github actions bot for commits
update benchmark workflow to use github actions bot as per https://github.com/actions/checkout/discussions/479#discussioncomment-625461

### PR DESCRIPTION
**Pull request by nerdvegas**
Closes #1318

### CODE REVIEW

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]
Looks great, just left a little suggestion.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,17 +21,18 @@ sending a pull request. Please follow these guidelines:
     document the setting. The comments in this file are extracted and turned into Wiki content. Pay
     attention to the comment formatting and follow the existing style closely.
 
-## Windows Docker Workflow
+## DCO
 
-The Windows tests currently build a Python image for each version to test. Each is based on a common
-base image. Any changes to the following Docker images sources should be a separate commit:
+Rez enforces Developer Certificate of Origin (DCO) on all commits, as per ASWF guidelines.
```
[the remark this answers is not in the document]
Comment (nerdvegas) [reply CONTRIBUTING.md:26]: What timing jc! Github now has native DCO support, and aswf has activated it as of yesterday: https://github.blog/changelog/2022-06-08-admins-can-require-sign-off-on-web-based-commits/.

I'll add this link instead, cheers

[the remark this answers is not in the document]
Comment (JeanChristopheMorinPerso) [reply CONTRIBUTING.md:26]: What they announced is slightly different though as it only applies to Web commits.

[the remark this answers is not in the document]
Comment (JeanChristopheMorinPerso) [reply CONTRIBUTING.md:26]: So I'd still link to the probot DCO page.

**Comment by nerdvegas** [comment]
*On web-based commits... I completely missed that, you're right. That's a
little underwhelming.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**